src/Utils.ps1

```diff
@@ -301,6 +301,20 @@ function Get-PromptPath {
     return $currentPath
 }
 
+$sshType = 'System.Management.Automation.Runspaces.SSHConnectionInfo' -as [Type]
+$sshUserName = Get-Runspace | ForEach-Object ConnectionInfo |
+  Where-Object { $sshType -and $_ -is $sshType } | ForEach-Object UserName
+
+function Get-PromptConnectionInfo {
+  if (Test-Path Env:SSH_CONNECTION) {
+    if ($sshUserName -and $sshUserName -ne [System.Environment]::UserName) {
```

Comment:
They made a late change in that PowerShell PR due to user names on Linux being case-sensitive.  I think you want `-cne` here.  Also, I thought we were using soft-tabs of *4* spaces?  :-)  I'll test this tomorrow when I get into work and have access to a system with `sshd` set to use PowerShell.